libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -144,6 +144,7 @@ export class Function extends cloud.FunctionBase {
       timeout: props.timeout
         ? props.timeout.seconds
         : Duration.fromMinutes(1).seconds,
+      memorySize: props.memory ? props.memory : undefined,
```

Comment:
Can we add tests showing what the expected behavior is when memory is configured on a cloud.Function for each cloud?